### PR TITLE
Simulation Improvements

### PR DESCRIPTION
**Pull request by Jeff300fang**
This change adds a temporary position controller to allow people to begin developing autonomy. This position controller may need to be changed in the future to perform better in real life and to formally handle traversal completions. This pull request also removes the lighting from the simulation and replaces it with a single light mounted on the submarine. This better immitates what the sub sees in real life. An attempt was made to also bring in the 3D model of tech triton as an STL file mesh into the simulation. However, this is on hold because this import has affected the dynamics of the sub and other parameters need to be more formally defined for it to work. These changes also include some improvements to the sim world to better reflect the pool and the prequal task. The angle of the sonoptix is also changed from 45 degrees to 25 degrees to better reflect the viewing angle of it on the sub. For now, the new sub xacro file has been consolidated to pontus_real.xacro

This Improves #37 

image

image

### CODE REVIEW

**Comment by chachmu** [comment]
Long term we might consider splitting the acrylic components off the mesh so we don't have the entire frame be transparent

File: pontus_controller/pontus_controller/command_pos.py
Comment (chachmu) [inline pontus_controller/pontus_controller/command_pos.py:1]: What is this file used for?

File: pontus_description/models/pontus/components/sensors.xacro [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <gazebo reference="camera_${camera_id}">
       <sensor name="camera_${camera_id}" type="camera">
         <camera>
-          <horizontal_fov>1.047</horizontal_fov>
+          <horizontal_fov>1.20428</horizontal_fov>
```
Comment (chachmu) [inline pontus_description/models/pontus/components/sensors.xacro:21]: Where did you get this number? It's definitely closer than the one I originally put but I'm not sure it matches the oakd

**Review by chachmu — APPROVED** [review]
Everything looks pretty good. I think we definitely need to tune the sim PID values though, it seems like it overshoots pretty badly in z

File: pontus_description/models/pontus/components/sensors.xacro (hunk 1 above)
Comment (Jeff300fang) [reply pontus_description/models/pontus/components/sensors.xacro:21]: https://shop.luxonis.com/products/oak-d?srsltid=AfmBOopSnIn7--DWv3Nyrx8JXD2pZj5haGXN7MH7RJr1MXEPkIy2Gezz 

Under technical specifications, it says 69 degrees = 1.20428 radians HFOV for the color camera.

File: pontus_controller/pontus_controller/command_pos.py
Comment (Jeff300fang) [reply pontus_controller/pontus_controller/command_pos.py:1]: Oh, lol I just used this to spawn the /cmd_pos topic so I could publish data to it using rqt. Didn't know rqt could just create topics for you until now.

**Pull request merged**